packages/clay-card/src/CardWithInfo.tsx

```diff
@@ -170,32 +170,34 @@ export const ClayCardWithInfo: React.FunctionComponent<IProps> = ({
 				/>
 			)}
 
-			<ClaySticker
-				displayType={
-					stickerProps && stickerProps.displayType
-						? stickerProps.displayType
-						: 'primary'
-				}
-				position="bottom-left"
-				{...stickerProps}
-			>
-				{stickerProps ? (
-					stickerProps.children ? (
-						stickerProps.children
+			{stickerProps && (
+				<ClaySticker
+					displayType={
+						stickerProps && stickerProps.displayType
+							? stickerProps.displayType
+							: 'primary'
+					}
+					position="bottom-left"
+					{...stickerProps}
+				>
+					{stickerProps ? (
+						stickerProps.children ? (
+							stickerProps.children
+						) : (
+							stickerProps.content
+						)
 					) : (
-						stickerProps.content
-					)
-				) : (
-					<ClayIcon
-						spritemap={spritemap}
-						symbol={
-							isCardType.image
-								? 'document-image'
-								: 'document-default'
-						}
-					/>
-				)}
-			</ClaySticker>
+						<ClayIcon
+							spritemap={spritemap}
+							symbol={
+								isCardType.image
+									? 'document-image'
+									: 'document-default'
+							}
+						/>
```

Comment:
@julien I think this guy will no longer be rendered after this change, we may have to improve this conditional if we still want to support this case.

Reply:
@matuzalemsteles exactly what we were talking about with @carloslancha :+1:

Reply:
Thanks @julien for the info.